docs/src/documentation/03-components/03-information/badge/03-accessibility.mdx

```diff
@@ -0,0 +1,35 @@
+---
```

Comment:
In the [Guidelines tab](https://marco-badge-a11y.orbit.pages.dev/components/information/badge/#support-with-icons), there is link to the [Accessibility article](https://marco-badge-a11y.orbit.pages.dev/foundation/accessibility/). Maybe it would make more sense to link there this Accessibility tab instead of the article?